include/wx/gtk/caret.h

```diff
@@ -0,0 +1,77 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/gtk/caret.h
+// Purpose:     wxCaret class - the GTK3 implementation of wxCaret
+// Author:      Kettab Ali
+// Created:     2022-01-29
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_GTK_CARET_H_
+#define _WX_GTK_CARET_H_
+
+#include "wx/timer.h"
+
+class WXDLLIMPEXP_CORE wxCaret : public wxCaretBase
+{
+public:
+    wxCaret() { Init(); }
+        // create the caret of given (in pixels) width and height and associate
+        // with the given window
+    wxCaret(wxWindow *window, int width, int height)
```

Comment:
Why use `wxWindow` here and `wxWindowBase` below. I see msw does it too, but the base and generic don't.

Reply:
Oops! this is a stupid copy/past error, corrected now.